docs/02-app/01-building-your-application/01-routing/08-route-groups.mdx

```diff
@@ -54,6 +55,20 @@ To opt specific routes into a layout, create a new route group (e.g. `(shop)`) a
   height="930"
 />
 
+### Opting loading skeletons to a specific route
+
+To opt a [loading skeleton](/docs/app/building-your-application/routing/loading-ui-and-streaming) via a `loading.js` file to a specific route, create a new route group (e.g., `/(overview)`) and then move your `loading.tsx` inside that route group.
```

Comment:
```suggestion
To apply a [loading skeleton](/docs/app/building-your-application/routing/loading-ui-and-streaming) via a `loading.js` file to a specific route, create a new route group (e.g., `/(overview)`) and then move your `loading.tsx` inside that route group.
```

Reply:
I agree that more should probably be said here to clarify why a route group is useful. Because you can apply a loading.js to a specific route just by creating a new route segment and adding a loading.js to it. It's specifically useful when you don't want to apply loading to a particular segment without affecting the route structure or other pages that are within that segment.

Reply:
@ztanner  

>   It's specifically useful when you want to apply loading to a particular segment **without affecting the route structure** or **other pages that are within that segment**.

The first bold part in your above statement is confusing me a bit. You can already do that no (without route groups)? Just copy the loading file into segment (the folder) where you need it and it will apply (notice **you did not affect the route structure** by this move). 

Now, the second bold part from your statement makes more sense to me. Because due to what I did in my above paragraph (copy the loading file to segment) the problem now is that this loading will apply also to the _children routes within that segment_. No? 

Anyway, I am myself not very familiar with Nextjs so I might not be of much value in further discussions. I can approve this review because it is better than nothing, but it could be better - especially if it highlights these points we are discussing in a clear way.